src/services/EdFi.DataManagementService.Api.Tests.E2E/Features/EqualityConstraintValidation107.feature.disabled

```diff
@@ -0,0 +1,126 @@
+Feature: Equality Constraint Validation
+
+        Background:
+            Given this school exists
+                  | schoolId  | nameOfInstitution      | gradeLevels     | educationOrganizationCategories |
+                  | 255901001 | Grand Bend High School | ["Ninth grade"] | ["School"]                      |
+
+        @ignore
+        Scenario: Making a Post request when value does not match the same value in an array
+             When a POST request is sent to "/ed-fi/sections" with body
+                  """
+                  {
+                      "sectionIdentifier": "25590100102Trad220ALG112011Test",
+                      "courseOfferingReference": {
+                          "localCourseCode": "ALG-1",
+                          "schoolId": 255901001,
+                          "schoolYear": 2022,
+                          "sessionName": "2021-2022 Fall Semester"
+                      },
+                      "classPeriods": [
+                          {
+                              "classPeriodReference": {
+                                  "classPeriodName": "02 - Traditional",
+                                  "schoolId": 255901107
+                              }
+                          }
+                      ]
+                  }
+                  """
+             Then it should respond with 400
+              And the response body is
+                  """
+                  {
+                      "detail": "Data validation failed. See 'validationErrors' for details.",
+                      "type": "urn:ed-fi:api:bad-request:data",
+                      "title": "Data Validation Failed",
+                      "status": 400,
+                      "correlationId": null,
+                      "validationErrors": {
+                          "$.classPeriods[0].classPeriodReference.schoolId": [
+                              "All values supplied for 'schoolId' must match. Review all references (including those higher up in the resource's data) and align the following conflicting values: '255901001', '255901107'"
+                          ]
+                      }
+                  }
+                  """
+
+
+
+        @ignore
+        Scenario: Making a Post request when a value matches the first scenario in an array but not the second
+             When a POST request is sent to "/ed-fi/sections" with body
```

Comment:
This should be When a POST request is made to "<url>" with

Reply:
Complete